PredixMobileUIKit/PredixTimeSeriesView.swift

```diff
@@ -13,30 +13,9 @@ import PredixSDK
 /// PredixTimeSeriesView -- TimeSeries chart built with `LineChartView`.
 @IBDesignable
 open class PredixTimeSeriesView: LineChartView {
-    ///A delegate to allow a class to know when it's time to load data into the view
+    /// A delegate to allow a class to know when it's time to load data into the view
     @IBOutlet
-    open weak var timeSeriesDataDelegate: TimeSeriesViewDelegate? {
-        didSet {
-            
-            /*if let dataFunction = timeSeriesDataDelegate?.loadTimeSeriesData {
-                self.showSpinner()
```

Comment:
The calls to ShowSpinner and HideSpinner have been removed, but the methods themselves are still in the code, unused, uncallable, and untested.... They should be removed.